modules/vulnerability-scores.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: CONCEPT
+[id="vulnerability-scores_{context}"]
+= Vulnerability scores
+
+[role="_abstract"]
+In the RHACS portal, {product-title} shows a single CVSS base score for each vulnerability.
```

Comment:
After reading through a few more of the PRs, I realise that the product name "Red Hat Advanced Cluster Security for Kubernetes" is always referred to in full in the docs and the portal for the product is always referred to as "the RHACS portal".

In hindsight, you could consider referring to the full name of the portal the first time you use it and then use the acronym for further references, in each module.

@bergerhoffer what is your view on this? Thanks!

Reply:
Yeah, as long as RHACS is an approved abbreviation, then I agree it might be better to try to use after the first instance written out, especially because it is a mouthful. But I think on this initial conversion, since I assume @gaurav-nelson is doing some find/replaces for the name, it probably isn't a priority to worry about right now. But definitely something to consider in the future.

Reply:
@bergerhoffer thank you for clarifying that.